js/events/core/events_engine.js

```diff
@@ -171,14 +171,22 @@ var getHandlersController = function(element, eventName) {
 
             var firstHandlerForTheType = eventData.handleObjects.length === 1;
             var shouldAddNativeListener = firstHandlerForTheType && eventNameIsDefined;
+            var options;
 
             if(shouldAddNativeListener) {
                 shouldAddNativeListener = !special.callMethod(eventName, "setup", element, [ data, namespaces, handler ]);
             }
 
             if(shouldAddNativeListener) {
                 eventData.nativeHandler = getNativeHandler(eventName);
-                eventData.removeListener = domAdapter.listen(element, NATIVE_EVENTS_TO_SUBSCRIBE[eventName] || eventName, eventData.nativeHandler);
+
+                if(eventName === "touchmove") { // T630650
```

Comment:
Is it the final or temporary solution? If it is final, there should not be an issue note. Maybe we need some constant variable to store "touchmove" string. So its name will explain why this event is so special.